AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLRequest/GraphQLRequest+AnyModelWithSync.swift

```diff
@@ -64,13 +64,13 @@ extension GraphQLRequest: ModelSyncGraphQLRequestFactory {
                                                    decodePath: document.name)
     }
 
-    public static func createMutation(of model: Model, version: Int?) -> GraphQLRequest<MutationSyncResult> {
```

Comment:
Why do we need these changes?

Reply:
I noticed in the integration tests over in AWSAPIPlugin that tests these paths, for some reason were not compiling. until i added the default parameters here. Not sure how those were written without the nil default parameter on these methods